CMakeLists.txt

```diff
@@ -17,35 +17,35 @@
 #   51 Franklin St., Fifth Floor, Boston, MA 02110, USA
 # *************************************************************************************/
 
-IF(APPLE)
-  # productbuild packaging stuff 
-  cmake_minimum_required(VERSION 3.23)
-ELSE()
-  cmake_minimum_required(VERSION 3.1)
-ENDIF()
+IF (APPLE)
```

Comment:
Do I understand correctly, that all changes in this file are some other formatting?